extensions/xcode/src/open-url-in-simulator.command.tsx

```diff
@@ -0,0 +1,37 @@
+import { XcodeSimulatorService } from "./services/xcode-simulator.service";
+import { Clipboard, closeMainWindow, showToast, Toast } from "@raycast/api";
+
+interface OpenSimulatorArguments {
+  url?: string;
+}
+
+export default async (props: { arguments: OpenSimulatorArguments }) => {
+  const toast = await showToast({
```

Comment:
Before showing a toast it would be good to check if an Xcode installation is available as the subsequent invocations are relying on `xcrun`. If Xcode is not installed a dedicated error message would be suitable to inform the user.